modules/apps/headless-apio/structured-content/structured-content-apio-api/src/main/java/com/liferay/structured/content/apio/architect/filter/Filter.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.structured.content.apio.architect.filter;
+
+import org.apache.olingo.server.api.uri.queryoption.expression.Expression;
```

Comment:
mmm we are coupling our API with an external library... I suppose that is the "one" that has everything we need :)